src/lib.rs

```diff
@@ -0,0 +1 @@
+// This is a dummy library to work around limitations with `cargo-release`.
```

Comment:
I'm cool with this as long as it doesn't mess up our IDE / ability to run cargo from the root. I'm just going to test it before approving